docs/getting-started-with-micro/index.md

```diff
@@ -3,8 +3,6 @@ title: "Using Snowplow Micro locally"
 sidebar_position: 25
 ---
 
-```mdx-code-block
-import DocCardList from '@theme/DocCardList';
+[Snowplow Micro](/docs/testing-debugging/snowplow-micro/what-is-micro/index.md) is a minified version of the Snowplow pipeline that can run on your machine. For a few quick tests, Micro can be a simple alternative to deploying an full-blown Snowplow Open Source pipeline to your cloud. However, note that Micro does not persist the data in any warehouse or database — you can just inspect it using the API while Micro is running.
```

Comment:
Nit: I think "you can only inspect it using the API while Micro is running" reads slightly better than "just" in this case.